### DISCUSSION TITLE
Freezing "Create list" Modal when the stage of "creating..."

### DISCUSSION THREAD

**Discussion by xuelink** [community/community#64614] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

# Summarize
It is freezing whenever i click "Created" button that is show in the following screenshot. After refreshing, i can see list item added but modal is freezing.

## Before you start
- It is a private repo (and self-starred)
- MacBook Pro, 14-inch, 2023,
   - Chip: Apple M2 Max
   - Memory: 32GB
   - macOS: 13.5 (22G74)
- Firefox Browser (116.0.2 (64-bit))

## Screenshots

<img width="1800" alt="Screenshot 2023-08-22 at 16 25 15" src="https://github.com/community/community/assets/3280475/935636b5-0475-4fe8-b8a2-2b47d58aa1b5">

<img width="904" alt="Screenshot 2023-08-22 at 16 24 14" src="https://github.com/community/community/assets/3280475/de9deab8-e752-4b3e-b970-c8ba1373b0c3">

<img width="338" alt="Screenshot 2023-08-22 at 16 33 32" src="https://github.com/community/community/assets/3280475/a82f01db-4fbf-4b29-85ba-2388c6d823d0">

## More
Feel free to comment if you have any questions or improvements

**Comment by samuelwarang** [comment]
After clicking on Share feedback and report the website is redirected but the lists pop up remains


![image](https://github.com/community/community/assets/92023869/da59b1de-fe9d-49af-9928-9385e35f33cf)

**Comment by ysh83737** [comment]
This issue occurs every time creating a list. This list can only be seen after refreshing.
<img width="586" alt="image" src="https://github.com/community/community/assets/32612643/ebec6c32-8589-4b79-be6f-2b0fe2a2baee">

**Comment by ysh83737** [comment]
Earlier same issue #58574

**Reply by xuelink** [reply]
but it didnt still solved

**Comment by rS7Y** [comment]
It freezes every damn time I make a new list
<img width="623" alt="rusty Google Chrome-08-28 at 21 05 28@2x" src="https://github.com/community/community/assets/20349989/da496d81-c474-45e1-94e5-abdd7b71fd07">
.

Not sure why it's not fixed yet.

**Comment by haelyons** [comment]
Experienced this issue for several weeks across browsers (Chrome, Edge) on Windows 10 (19044.3086), though it now seems to be fixed (also across browsers). Can someone confirm? May have been stopped by:
1. Adding a repository to a new list, and reloading to allow the list to be created (as described by previous posts)
2. Un-starring the same repo without deleting the list
3. Adding the repo back to a new list via the same menu (reload unnecessary).

**Reply by xuelink** [reply]
star-unstar without deleting and star it again. yes it could work, but it is a bug that has to be committed and to be solved!

**Reply by haelyons** [reply]
These are steps which may help in identifying and reproducing the cause of the issue as well as a potential fix, not a long-term solution.

**Reply by musabadru** [reply]
All the above issues still present in Opera Mobile v77. Popup persisted even while i came to report the bug

**Comment by ghost** [comment]
**Same issue facing. When GitHub will solve it?**

**Comment by fecony** [comment]
`https://github.com/stars/<username>/lists` returns 200 but UI is not updating, It should be 5 minute fix but It's happening ever since lists appeared 😄

**Comment by vitaliizghonnik** [comment]
It would be greatly appreciated if the team could allocate more attention to this issue.

I encountered a bug in the beta version where the loading screen persists indefinitely when trying to create a new list in the ‘Your Star’ section. Here are the steps to reproduce the issue:

1. Add a repository to the ‘Your Star’ section.
2. Open the ‘Your Star’ section.
3. Click on the ‘Starred’ dropdown menu of the chosen repository.
4. From the popped-out ‘Lists’ menu, choose ‘Create list’.
5. In the field with placeholder text ‘Name this list’, enter the name of the list.
6. Click the ‘Create’ button.

Expected Result: The new list should be created and the loading screen should disappear.
Actual Result: The loading screen displays ‘Creating…’ for an infinite amount of time.
The process is repetitive each time.
Browser - Chrome Version 117.0.5938.150
![Create New List via Starred dropdown menu](https://github.com/community/community/assets/115250975/996c3b34-4d23-497b-b2ad-e1e696c3a2d6)

Please look into this issue. Thank you!

**Reply by inilord** [reply]
Great

**Comment by EVERCLEAR11** [comment]
> ### Select Topic Area
> 
> Bug
> 
> ### Body
> 
> # Summarize
> It is freezing whenever i click "Created" button that is show in the following screenshot. After refreshing, i can see list item added but modal is freezing.
> 
> ## Before you start
> - It is a private repo (and self-starred)
> - MacBook Pro, 14-inch, 2023,
>    - Chip: Apple M2 Max
>    - Memory: 32GB
>    - macOS: 13.5 (22G74)
> - Firefox Browser (116.0.2 (64-bit))
> 
> ## Screenshots
> 
> <img width="1800" alt="Screenshot 2023-08-22 at 16 25 15" src="https://github.com/community/community/assets/3280475/935636b5-0475-4fe8-b8a2-2b47d58aa1b5">
> 
> <img width="904" alt="Screenshot 2023-08-22 at 16 24 14" src="https://github.com/community/community/assets/3280475/de9deab8-e752-4b3e-b970-c8ba1373b0c3">
> 
> <img width="338" alt="Screenshot 2023-08-22 at 16 33 32" src="https://github.com/community/community/assets/3280475/a82f01db-4fbf-4b29-85ba-2388c6d823d0">
> 
> ## More
> Feel free to comment if you have any questions or improvements
> 
>

**Comment by EVERCLEAR11** [comment]
> > ### Select Topic Area
> > 
> > Bug
> > 
> > ### Body
> > 
> > # Summarize
> > It is freezing whenever i click "Created" button that is show in the following screenshot. After refreshing, i can see list item added but modal is freezing.
> > 
> > ## Before you start
> > - It is a private repo (and self-starred)
> > - MacBook Pro, 14-inch, 2023,
> >    - Chip: Apple M2 Max
> >    - Memory: 32GB
> >    - macOS: 13.5 (22G74)
> > - Firefox Browser (116.0.2 (64-bit))
> > 
> > ## Screenshots
> > 
> > <img width="1800" alt="Screenshot 2023-08-22 at 16 25 15" src="https://github.com/community/community/assets/3280475/935636b5-0475-4fe8-b8a2-2b47d58aa1b5">
> > 
> > <img width="904" alt="Screenshot 2023-08-22 at 16 24 14" src="https://github.com/community/community/assets/3280475/de9deab8-e752-4b3e-b970-c8ba1373b0c3">
> > 
> > <img width="338" alt="Screenshot 2023-08-22 at 16 33 32" src="https://github.com/community/community/assets/3280475/a82f01db-4fbf-4b29-85ba-2388c6d823d0">
> > 
> > ## More
> > Feel free to comment if you have any questions or improvements
> > 
> > 
> 
>

**Comment by tgrecojs** [comment]
I opened a similar issue here:

https://github.com/orgs/community/discussions/76972#discussion-5889632

**Comment by Dsbryant** [comment]
An answer

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.

**Comment by Al3676** [comment]
Suggest
